### ISSUE TITLE
Docker: chifra init doesn't use env vars correctly

### ISSUE THREAD

**Issue by quickchase** [TrueBlocks/trueblocks-core#2060]
I'm setting the `SETTINGS_RPCPROVIDER` variable as described [HERE](https://github.com/TrueBlocks/trueblocks-core/blob/master/docs/content/docs/chifra/configs.md)
```
export SETTINGS_RPCPROVIDER=https://rpc.archivenode.io/<redacted>/
```

This works when I run `./chifra status`, it talks to the endpoint that I have configured above. However, when I run `./chifra init` it tries to talk to `http://localhost:8545` (which is what's in the `trueBlocks.toml` file by default).

If I update the `trueBlocks.toml` variable, it works fine but I think the ENV variables should take preference here.

Additionally, `./chifra init` seems to ALWAYS talk to `https://ipfs.unchainedindex.io/ipfs` regardless of how I set `ipfs_gateway` it doesn't seem to honor either the ENV variable (`SETTINGS_IPFS_GATEWAY`) OR the `trueBlocks.toml` file.

**Comment by tjayrush** [comment]
Are you on the multi-chain-11 branch?  I don’t think the env variable will work yet, but just wondering. 

--------
Thomas Jay Rush
***@***.***

**Comment by quickchase** [comment]
No, it looks like I built off `master` - what branch or release should I be working off? `develop`? `v0.23.7.0-alpha`?

What is the ideal candidate for a DAppNode package release?

**Comment by tjayrush** [comment]
We are very close to finishing the multi-chain-11 branch. When that’s finished it will merge into develop and then master. Once that happens, we’re going to call ourselves beta. 

I will make sure these env variable issues are fixed before that happens, so maybe docker can happen from master once multi chain is ready. 

The current master is probably at least a month behind the multi chain stuff, so maybe it’s best not to use that. 

--------
Thomas Jay Rush
***@***.***

**Comment by MysticRyuujin** [comment]
Sounds good! I'll keep hacking with the assumption that ENV will be fixed and plan on a release to beta once it's ready!

**Comment by quickchase** [comment]
I think in the `multi-chain-11` it's even more messed up...

```
% grep rpcProvider /Users/chase/Library/Application\ Support/TrueBlocks/trueBlocks.toml
# The 'rpcProvider' value points to a valid RPC endpoint. TrueBlocks will not
rpcProvider = "https://rpc.archivenode.io/<redacted>/"
```

```
% ~/github/trueblocks-core/bin/chifra status

  Warning: The RPC server (https://) was not found. Either start it, or edit the
  rpcProvider value in the file /Users/chase/Library/Application Support/TrueBlocks/trueBlocks.toml. Quitting...
```

AND it's ignoring my ENV variable

**Comment by tjayrush** [comment]
Can you run through this migration? https://github.com/TrueBlocks/trueblocks-core/blob/multi-chain-11/src/other/migrations/README-v0.25.0.md

**Comment by MysticRyuujin** [comment]
I just deleted it and started over lol I'm good now, I think. Except for the stuff I sent you on Discord. But I'll play with it some more.

**Comment by tjayrush** [comment]
This is fixed in the latest "official" docker version
